synctl/cli.py

```diff
@@ -435,6 +443,18 @@ def get_browser_script_test(self):
                     print("frequency is not valid, it should be in [1,120]")
         return browser_script_test
 
+    def get_ssl_test(self):
+        ssl_test = {}
+        ssl_test["testCount"] = int(self.ask_question("How many ISM tests (SSLCertificate) do you want to create? (0 if no) "))
+        if ssl_test["testCount"] > 0:
+            while True:
+                ssl_test["frequency"] = int(self.ask_question("What is the test frequency for SSLCertificate tests? (1-1440) "))
```

Comment:
keep consistency as last question, What is the test frequency for ISM test (SSLCertificate)? (1-1440)